src/shared/tpm2-util.c

```diff
@@ -1555,22 +1572,39 @@ static int openssl_pubkey_to_tpm2_pubkey(EVP_PKEY *input, TPM2B_PUBLIC *output)
         if (BN_bn2bin(e, (unsigned char*) &output->publicArea.parameters.rsaDetail.exponent) <= 0)
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Failed to convert RSA exponent.");
 
+        if (ret_fp) {
+                _cleanup_free_ void *fp = NULL;
+                size_t fp_size;
+                int r;
```

Comment:
we always have `r` with the initial function variable declarations, even if it's only used in a block, as a special case

Reply:
ack, updated, thnx